### PR TITLE
Docs(Other): Fix environment variable name for setting lambda URL

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Current docker-compose.yml snippet has incorrect env variable. As a result, the lambda server URL is ignored. The new syntax was introduced in: https://github.com/dgraph-io/dgraph/commit/12c3ef564cde11ecc3de96ec1516b3148e52d795

This PR fixes this issue.

### CODE REVIEW

**Comment by MichelDiz** [comment]
Hey @mszostok I'm not able to verify this env variable. Where did you get it from?

**Comment by mszostok** [comment]
Oh, this PR is 8 months old 😄 not sure if this still applies, it was a bug back then. I don't use the dgraph anymore, so I will just close this PR. 

If you go to the commit that I mentioned in the PR description, then you will see this change in file [`dgraph/cmd/alpha/run.go`](https://github.com/dgraph-io/dgraph/commit/12c3ef564cde11ecc3de96ec1516b3148e52d795#diff-a72445362e52218163e2d2f572b946ff3aea61f8675aa235a5dab3227ac6168e):

Screen Shot 2022-10-19 at 19 31 57

TL;DR; what you can see is that the `lambda-url` was previously defined under `graphql` so it was translated to `DGRAPH_ALPHA_GRAPHQL: "lambda-url=http://dgraph_lambda:8686/graphql-worker"`

but the PR changed it from `lambda-url` to `url` and nest it under `lambda` so it translates to `DGRAPH_ALPHA_LAMBDA: "url=http://dgraph_lambda:8686/graphql-worker"`


-- 
I see that dgraph switched to `main` but also somehow lost the history from `master` 🤔 so maybe that's not an issue anymore 😄
